src/main/java/com/example/onlinebookstore/mapper/BookMapperImpl.java

```diff
@@ -0,0 +1,74 @@
+package com.example.onlinebookstore.mapper;
+
+import com.example.onlinebookstore.dto.BookDto;
+import com.example.onlinebookstore.dto.CreateBookRequestDto;
+import com.example.onlinebookstore.model.Book;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BookMapperImpl implements BookMapper {
```

Comment:
this class should be generated in target folder

Reply:
then checkstyle test will fail

Reply:
It is necessary to exclude the check of the target directory from the checkstyle.
Write to slack, they will help you how to do it.